docs/pages/toolpad/core/api/app-provider.json

```diff
@@ -12,6 +12,12 @@
       },
       "default": "null"
     },
+    "localeText": {
+      "type": {
+        "name": "shape",
+        "description": "{ accountPreviewIconButtonLabel?: string, accountPreviewTitle?: string, accountSignInLabel?: string, accountSignOutLabel?: string, alert?: string, cancel?: string, close?: string, confirm?: string, delete?: string, email?: string, loading?: string, magicLinkSignInTitle?: string, oauthSignInTitle?: string, ok?: string, or?: string, passkey?: string, passkeySignInTitle?: string, password?: string, save?: string, signInRememberMe?: string, signInSubtitle?: string, signInTitle?: string, to?: string, username?: string, with?: string }"
```

Comment:
This won't scale. Any way we can do the same as [X](https://github.com/mui/mui-x/blob/b252b9a9562a69512b74d3cb89049be5591d908b/docs/pages/x/api/date-pickers/static-time-picker.json#L15)?